common-ui/src/main/res/values/design-system-dimensions.xml

```diff
@@ -44,6 +44,8 @@
     <dimen name="oneLineItemPadding">14dp</dimen>
     <dimen name="twoLineItemHeight">56dp</dimen>
     <dimen name="twoLineItemPadding">14dp</dimen>
+    <dimen name="listItemSectionHeaderIconSize">24dp</dimen>
```

Comment:
I think I also applied this dimen to the `DaxTextView` because the text was cut off vertically in some cases. I fixed that setting a height to the `LinearLayout` instead 👍